### ISSUE TITLE
[Bug]: JavaScript heap out of memory

### ISSUE THREAD

**Issue by kiliter** [advplyr/audiobookshelf#3150]
### What happened?

1. run as synology Nas 
2. play a book on ios then i closed 
3. next day my audiobookshelf was out of memory 

### What did you expect to happen?

JavaScript heap out of memory

### Steps to reproduce the issue

1. play with ios 
2. close app
3. wait next day
4. JavaScript heap out of memory

### Audiobookshelf version

2.11.0

### How are you running audiobookshelf?

Other (list in "Additional Notes" box)

### What OS is your Audiobookshelf server hosted from?

Other (list in "Additional Notes" box)

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

None

### Logs

```shell
Tue Jul  9 17:52:10 CST 2024
Starting Audiobookshelf command ./audiobookshelf
production Config /var/packages/Audiobookshelf/target/config /var/packages/Audiobookshelf/target/metadata
[2024-07-09 17:52:13.877] INFO: === Starting Server ===
[2024-07-09 17:52:13.878] INFO: [Server] Init v2.11.0
[2024-07-09 17:52:13.878] INFO: [Server] Node.js Version: v20.11.1
[2024-07-09 17:52:55.062] INFO: [Database] Initializing db at "/var/packages/Audiobookshelf/target/config/absdatabase.sqlite"
[2024-07-09 17:52:55.151] INFO: [Database] Db connection was successful
[2024-07-09 17:52:55.492] INFO: [Database] Db initialized with models: user, library, libraryFolder, book, podcast, podcastEpisode, libraryItem, mediaProgress, series, bookSeries, author, bookAuthor, collection, collectionBook, playlist, playlistMediaItem, device, playbackSession, feed, feedEpisode, setting, customMetadataProvider, mediaItemShare
[2024-07-09 17:52:55.632] INFO: [LogManager] Init current daily log filename: 2024-07-09.txt
[2024-07-09 17:53:05.564] ERROR: [FileUtils] ReadTextFile error Error: Cannot create a string longer than 0x1fffffe8 characters (    at Object.readTextFile (/snapshot/audiobookshelf/server/utils/fileUtils.js))
[2024-07-09 17:53:05.666] INFO: [BackupManager] 5 Backups Found
[2024-07-09 17:53:05.695] INFO: [Watcher] Initializing watcher for "有声小说".
Warning: connect.session() MemoryStore is not
designed for a production environment, as it will leak
memory, and will not scale past a single process.
[2024-07-09 17:53:05.751] INFO: Listening on http://0.0.0.0:13378
[2024-07-09 17:53:08.225] INFO: [Watcher] "有声小说" Ready
[2024-07-09 17:54:14.845] INFO: [SocketAuthority] Socket Connected _fbQq4NThzlh-2TaAAAB
[2024-07-09 17:55:41.008] INFO: [SocketAuthority] Socket _fbQq4NThzlh-2TaAAAB disconnected from client "KILITER" after 86163ms (Reason: transport close)
[2024-07-09 17:55:41.667] INFO: [SocketAuthority] Socket Connected _nmrEzPWse8rOlV0AAAD
[2024-07-09 17:56:38.699] INFO: [SocketAuthority] Socket _nmrEzPWse8rOlV0AAAD disconnected from client "KILITER" after 57032ms (Reason: transport close)
[2024-07-10 01:30:00.800] INFO: [BackupManager] Running Backup
[2024-07-10 01:30:02.719] INFO: [BackupManager] 1153025 total bytes
[2024-07-10 01:30:02.723] INFO: [BackupManager] Backup successful 2024-07-10T0130
[2024-07-10 01:30:02.731] INFO: [BackupManager] Backup "2024-07-05T0130" Removed
[2024-07-10 16:54:37.659] INFO: [SocketAuthority] Socket Connected KIH_6Qq5uaJJ-r2eAAAF
[2024-07-10 16:54:38.512] INFO: [PlaybackSessionManager] Syncing local session "一念永恒｜0001-0350｜" (25e93a04-4935-4df2-93c1-c800da012c1e)
[2024-07-10 16:54:38.903] INFO: [PlaybackSessionManager] Syncing local session "一念永恒｜0001-0350｜" (8844ad54-7647-407a-a3e5-f8e7a6574cb8)
[2024-07-10 16:54:55.750] INFO: [PlaybackSessionManager] startSession: Closing open session "一念永恒｜0001-0350｜" for user "KILITER" (Device: iPhone13,2 / v0.9.74)
[2024-07-10 16:54:55.974] INFO: [STREAM] START STREAM - Num Segments: 51553
[2024-07-10 16:54:56.062] INFO: [STREAM] Starting Stream at startTime 9:41.0 (User startTime 10:11.0) and Segment #96
[2024-07-10 16:54:56.596] INFO: [INFO] FFMPEG transcoding started with command: ffmpeg -seek_timestamp 1 -f concat -safe 0 -ss 581.2s -noaccurate_seek -i /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/files.txt -y -loglevel error -map 0:a -c:a aac -f hls -copyts -avoid_negative_ts make_non_negative -max_delay 5000000 -max_muxing_queue_size 2048 -hls_time 6 -hls_segment_type mpegts -start_number 96 -hls_playlist_type vod -hls_list_size 0 -hls_allow_cache 0 -hls_segment_filename /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-%d.ts /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/final-output.m3u8
[2024-07-10 16:54:56.597] INFO: 
[2024-07-10 16:54:58.600] INFO: [STREAM-CHECK] Check Files 6 of 51553 0.01% Furthest Segment: 101
[2024-07-10 16:55:00.606] INFO: [STREAM] 0a2b2464-0983-4e05-8a0b-c724d4747995 notifying client that stream is ready
[2024-07-10 16:55:00.633] INFO: [STREAM-CHECK] Check Files 12 of 51553 0.02% Furthest Segment: 107
[2024-07-10 16:55:02.606] INFO: [STREAM-CHECK] Check Files 20 of 51553 0.04% Furthest Segment: 115
[2024-07-10 16:55:04.607] INFO: [STREAM-CHECK] Check Files 27 of 51553 0.05% Furthest Segment: 122
[2024-07-10 16:55:06.607] INFO: [STREAM-CHECK] Check Files 34 of 51553 0.07% Furthest Segment: 129
[2024-07-10 16:55:08.608] INFO: [STREAM-CHECK] Check Files 40 of 51553 0.08% Furthest Segment: 135
[2024-07-10 16:55:10.611] INFO: [STREAM-CHECK] Check Files 48 of 51553 0.09% Furthest Segment: 143
[2024-07-10 16:55:12.609] INFO: [STREAM-CHECK] Check Files 56 of 51553 0.11% Furthest Segment: 151
[2024-07-10 16:55:12.812] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-0.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:12.813] WARN: [STREAM] Segment #0 Request is before starting segment number #96 - Reset Transcode (    at Stream.checkSegmentNumberRequest (/snapshot/audiobookshelf/server/objects/Stream.js))
[2024-07-10 16:55:12.831] INFO: [FFMPEG] Transcode Killed
[2024-07-10 16:55:13.314] INFO: Stream Reset New Start Time 0:00.0
[2024-07-10 16:55:13.316] INFO: [STREAM] START STREAM - Num Segments: 51553
[2024-07-10 16:55:13.331] INFO: [INFO] FFMPEG transcoding started with command: ffmpeg -seek_timestamp 1 -f concat -safe 0 -i /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/files.txt -y -loglevel error -map 0:a -c:a aac -f hls -copyts -avoid_negative_ts make_non_negative -max_delay 5000000 -max_muxing_queue_size 2048 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_playlist_type vod -hls_list_size 0 -hls_allow_cache 0 -hls_segment_filename /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-%d.ts /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/final-output.m3u8
[2024-07-10 16:55:13.332] INFO: 
[2024-07-10 16:55:13.400] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-1.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.401] INFO: [HlsRouter] Stream 0a2b2464-0983-4e05-8a0b-c724d4747995 is currently resetting
[2024-07-10 16:55:13.471] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-2.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.471] INFO: [HlsRouter] Stream 0a2b2464-0983-4e05-8a0b-c724d4747995 is currently resetting
[2024-07-10 16:55:13.536] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-3.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.537] INFO: [HlsRouter] Stream 0a2b2464-0983-4e05-8a0b-c724d4747995 is currently resetting
[2024-07-10 16:55:13.613] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-4.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.617] INFO: [HlsRouter] Stream 0a2b2464-0983-4e05-8a0b-c724d4747995 is currently resetting
[2024-07-10 16:55:13.701] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-5.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.703] INFO: [HlsRouter] Stream 0a2b2464-0983-4e05-8a0b-c724d4747995 is currently resetting
[2024-07-10 16:55:13.847] INFO: [STREAM] Clearing isResetting
[2024-07-10 16:55:13.853] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-6.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:13.957] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-7.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.046] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-8.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.134] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-9.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.208] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-10.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.281] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-11.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.416] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-12.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.482] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-13.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.527] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-14.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.609] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-15.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.669] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-16.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.727] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-17.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.798] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-18.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.856] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-19.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.918] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-20.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:14.982] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-21.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.050] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-22.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.130] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-23.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.181] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-24.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.228] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-25.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.283] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-26.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.355] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-27.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.398] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-28.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.451] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-29.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.517] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-30.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.587] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-31.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.647] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-32.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.714] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-33.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.809] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-34.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.854] INFO: [STREAM-CHECK] Check Files 64 of 51553 0.12% Furthest Segment: 152
[2024-07-10 16:55:15.887] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-35.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.946] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-36.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:15.994] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-37.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.042] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-38.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.107] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-39.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.166] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-40.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.235] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-41.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.308] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-42.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.383] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-43.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.449] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-44.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.507] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-45.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.566] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-46.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.626] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-47.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.690] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-48.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.771] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-49.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.832] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-50.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.922] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-51.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:16.987] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-52.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.047] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-53.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.118] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-54.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.179] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-55.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.256] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-56.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.306] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-57.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.366] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-58.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.428] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-59.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.486] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-60.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.540] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-61.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.617] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-62.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.701] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-63.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.768] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-64.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.829] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-65.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.854] INFO: [STREAM-CHECK] Check Files 72 of 51553 0.14% Furthest Segment: 152
[2024-07-10 16:55:17.878] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-66.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:17.948] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-67.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.008] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-68.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.062] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-69.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.137] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-70.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.192] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-71.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.339] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-72.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.408] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-73.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.468] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-74.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.547] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-75.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.606] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-76.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.717] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-77.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.857] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-78.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:18.991] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-79.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.096] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-80.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.157] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-81.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.235] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-82.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.295] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-83.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.369] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-84.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.460] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-85.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.529] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-86.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.615] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-87.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.700] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-88.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.842] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-89.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:19.855] INFO: [STREAM-CHECK] Check Files 79 of 51553 0.15% Furthest Segment: 152
[2024-07-10 16:55:20.174] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-90.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:20.222] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-91.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:20.274] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-92.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:20.327] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-93.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:20.406] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-94.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:20.486] WARN: File path does not exist /var/packages/Audiobookshelf/target/metadata/streams/0a2b2464-0983-4e05-8a0b-c724d4747995/output-95.ts (    at HlsRouter.streamFileRequest (/snapshot/audiobookshelf/server/routers/HlsRouter.js))
[2024-07-10 16:55:21.855] INFO: [STREAM-CHECK] Check Files 86 of 51553 0.17% Furthest Segment: 152
[2024-07-10 16:55:23.864] INFO: [STREAM-CHECK] Check Files 93 of 51553 0.18% Furthest Segment: 152
[2024-07-10 16:55:25.854] INFO: [STREAM-CHECK] Check Files 100 of 51553 0.19% Furthest Segment: 152
[2024-07-10 16:55:27.854] INFO: [STREAM-CHECK] Check Files 107 of 51553 0.21% Furthest Segment: 152
[2024-07-10 16:55:29.855] INFO: [STREAM-CHECK] Check Files 114 of 51553 0.22% Furthest Segment: 152
[2024-07-10 16:55:31.855] INFO: [STREAM-CHECK] Check Files 121 of 51553 0.23% Furthest Segment: 152
[2024-07-10 16:55:33.856] INFO: [STREAM-CHECK] Check Files 127 of 51553 0.25% Furthest Segment: 152
[2024-07-10 16:55:35.855] INFO: [STREAM-CHECK] Check Files 132 of 51553 0.26% Furthest Segment: 152
[2024-07-10 16:55:37.860] INFO: [STREAM-CHECK] Check Files 140 of 51553 0.27% Furthest Segment: 152
[2024-07-10 16:55:39.864] INFO: [STREAM-CHECK] Check Files 148 of 51553 0.29% Furthest Segment: 152
[2024-07-10 16:55:41.861] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:43.861] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:45.861] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:47.862] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:49.862] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:51.862] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:53.862] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:55.863] INFO: [STREAM-CHECK] Check Files 153 of 51553 0.30% Furthest Segment: 152
[2024-07-10 16:55:57.862] INFO: [STREAM-CHECK] Check Files 156 of 51553 0.30% Furthest Segment: 155
[2024-07-10 16:55:59.862] INFO: [STREAM-CHECK] Check Files 164 of 51553 0.32% Furthest Segment: 163
[2024-07-10 16:56:01.864] INFO: [STREAM-CHECK] Check Files 171 of 51553 0.33% Furthest Segment: 170
[2024-07-10 16:56:03.870] INFO: [STREAM-CHECK] Check Files 178 of 51553 0.35% Furthest Segment: 177
[2024-07-10 16:56:05.867] INFO: [STREAM-CHECK] Check Files 184 of 51553 0.36% Furthest Segment: 183
[2024-07-10 16:56:07.876] INFO: [STREAM-CHECK] Check Files 192 of 51553 0.37% Furthest Segment: 191
[2024-07-10 16:56:09.875] INFO: [STREAM-CHECK] Check Files 199 of 51553 0.39% Furthest Segment: 198
[2024-07-10 16:56:11.874] INFO: [STREAM-CHECK] Check Files 206 of 51553 0.40% Furthest Segment: 205
[2024-07-10 16:56:13.884] INFO: [STREAM-CHECK] Check Files 213 of 51553 0.41% Furthest Segment: 212
[2024-07-10 16:56:15.884] INFO: [STREAM-CHECK] Check Files 220 of 51553 0.43% Furthest Segment: 219
[2024-07-10 16:56:17.892] INFO: [STREAM-CHECK] Check Files 226 of 51553 0.44% Furthest Segment: 225
[2024-07-10 16:56:19.883] INFO: [STREAM-CHECK] Check Files 233 of 51553 0.45% Furthest Segment: 232
[2024-07-10 16:56:21.884] INFO: [STREAM-CHECK] Check Files 241 of 51553 0.47% Furthest Segment: 240
[2024-07-10 16:56:23.892] INFO: [STREAM-CHECK] Check Files 249 of 51553 0.48% Furthest Segment: 248
[2024-07-10 16:56:25.907] INFO: [STREAM-CHECK] Check Files 257 of 51553 0.50% Furthest Segment: 256
[2024-07-10 16:56:27.898] INFO: [STREAM-CHECK] Check Files 265 of 51553 0.51% Furthest Segment: 264
[2024-07-10 16:56:29.900] INFO: [STREAM-CHECK] Check Files 273 of 51553 0.53% Furthest Segment: 272
[2024-07-10 16:56:31.900] INFO: [STREAM-CHECK] Check Files 279 of 51553 0.54% Furthest Segment: 278
[2024-07-10 16:56:33.904] INFO: [STREAM-CHECK] Check Files 285 of 51553 0.55% Furthest Segment: 284
[2024-07-10 16:56:35.909] INFO: [STREAM-CHECK] Check Files 292 of 51553 0.57% Furthest Segment: 291
[2024-07-10 16:56:37.901] INFO: [STREAM-CHECK] Check Files 300 of 51553 0.58% Furthest Segment: 299
[2024-07-10 16:56:39.909] INFO: [STREAM-CHECK] Check Files 307 of 51553 0.60% Furthest Segment: 306
[2024-07-10 16:56:41.908] INFO: [STREAM-CHECK] Check Files 314 of 51553 0.61% Furthest Segment: 313
[2024-07-10 16:56:43.917] INFO: [STREAM-CHECK] Check Files 321 of 51553 0.62% Furthest Segment: 320
[2024-07-10 16:56:45.914] INFO: [STREAM-CHECK] Check Files 328 of 51553 0.64% Furthest Segment: 327
[2024-07-10 16:56:47.420] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.420] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.421] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.421] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.421] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.421] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.422] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.422] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.422] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.422] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.423] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.423] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 16:56:47.423] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.423] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 17:08:18.802] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 17:08:18.802] INFO: Error while decoding stream #0:0: Invalid data found when processing input
[2024-07-10 17:08:18.802] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 17:08:18.803] INFO: Error while decoding stream #0:0: Invalid data found when processing input

<--- Last few GCs --->

[2408:0x4d1ad40] 83765185 ms: Mark-Compact 4039.0 (4133.7) -> 4023.8 (4133.9) MB, 1848.61 / 0.00 ms  (average mu = 0.186, current mu = 0.095) task; scavenge might not succeed
[2408:0x4d1ad40] 83767186 ms: Mark-Compact 4039.5 (4134.0) -> 4024.8 (4135.0) MB, 1821.37 / 0.00 ms  (average mu = 0.142, current mu = 0.090) task; scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----

 1: 0xad2170 node::Abort() [./audiobookshelf]
 2: 0x9be40a  [./audiobookshelf]
 3: 0x155c490 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [./audiobookshelf]
 4: 0x155c777 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [./audiobookshelf]
 5: 0x176f8f5  [./audiobookshelf]
 6: 0x176fe84 v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [./audiobookshelf]
 7: 0x178668b v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::internal::GarbageCollectionReason, char const*) [./audiobookshelf]
 8: 0x1786e39 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [./audiobookshelf]
 9: 0x17dfb3c v8::internal::MinorGCJob::Task::RunInternal() [./audiobookshelf]
10: 0xb536b3  [./audiobookshelf]
11: 0xb53ad4 node::PerIsolatePlatformData::FlushForegroundTasksInternal() [./audiobookshelf]
12: 0xc81103  [./audiobookshelf]
13: 0xc97c4a  [./audiobookshelf]
14: 0x1526a20  [./audiobookshelf]
15: 0xa0cc06 node::SpinEventLoopInternal(node::Environment*) [./audiobookshelf]
16: 0xb25007 node::NodeMainInstance::Run() [./audiobookshelf]
17: 0xa963f2 node::Start(int, char**) [./audiobookshelf]
18: 0x7fbe667f6177  [/lib64/libc.so.6]
19: 0x7fbe667f6235 __libc_start_main [/lib64/libc.so.6]
20: 0x9fd03c  [./audiobookshelf]
```

### Additional Notes

[2024-07-10 16:56:47.422] INFO: [wmav2 @ 0x69132c0] Input packet size too small (371 < 743)
[2024-07-10 16:56:47.422] INFO: Error while decoding stream #0:0: Invalid data found when processing input
in log file with 400MB

**Comment by connoralydon** [comment]
do you have to restart your system to start it, or can you restart the application? if you restart the application does it end up in a global slowdown of the machine?

**Comment by mpclepto** [comment]
It appears the original poster has vanished, yet I believe I am having the same error code. I don't want to be a nusance however and tag mine in here unless you think I should? 

I get this:

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory

(I can provide more information here or open a fresh issue if you would like)

My ABS is crashing ~2-3 times a day.

**Comment by nichwall** [comment]
> It appears the original poster has vanished, yet I believe I am having the same error code.

Are you getting the same error with all of the transcoding/stream issues? If not, opening a new issue will probably be easier if you can include crash or other logs, along with what you may be doing when the crash happens (automatic library scans, downloading podcast episodes, server backup, using web client/Android/iOS, etc)
